libs/ui/src/lib/carousel/carousel.ts

```diff
@@ -0,0 +1,162 @@
+import {
+    AfterContentInit,
+    Component,
+    ContentChildren,
+    Directive,
+    ElementRef,
+    HostBinding,
+    Inject,
+    Input,
+    Optional,
+    QueryList,
+    ViewChild,
+    ViewEncapsulation,
+  } from '@angular/core';
+  import {FocusableOption, FocusKeyManager} from '@angular/cdk/a11y';
+  import {LEFT_ARROW, RIGHT_ARROW, TAB} from '@angular/cdk/keycodes';
+  import {ANIMATION_MODULE_TYPE} from '@angular/platform-browser/animations';
+  import {MatIconModule} from '@angular/material/icon';
+  import {MatButtonModule} from '@angular/material/button';
+  import {NgIf} from '@angular/common';
+  
+  @Directive({
```

Comment:
Can you extract this part to new file `carousel-item.directive.ts`?

Reply:
> only I have to copy that 4 lines and paste that into `carousel-item.directive.ts`?

No, from line 22 to line 35

Reply:
done this.
please check and let me know.

Reply:
Thanks 👍🏻